.changeset/empty-islands-itch.md

```diff
@@ -0,0 +1,6 @@
+---
+"@ultraviolet/form": patch
+"@ultraviolet/ui": patch
+---
+
+New components `SelectableCardGroup` in ultraviolet/ui and `SelectableCardGroupField` in ultraviolet/form
```

Comment:
You can split the changeset in 2 files, one for ui and one for form with the correct component name for each